core/src/main/java/org/apache/gravitino/storage/relational/service/RoleMetaService.java

```diff
@@ -109,35 +94,18 @@ public List<RoleEntity> listRolesByMetadataObjectIdentAndType(
                     metadataObjectId, metadataObject.type().name()));
     return rolePOs.stream()
         .map(
-            po ->
-                POConverters.fromRolePO(
-                    po, listSecurableObjects(po), AuthorizationUtils.ofRoleNamespace(metalake)))
+            po -> {
+              if (allFields) {
```

Comment:
When will you set this to `true` for list roles by metadata object?

Reply:
This is used for internal implement. I used it in the FutureGrantManager, I need find all the roles contains metalake metadata object.  For example, I find the role contains `CREATE TABLE` for a metalake and apply the role for newly created catalog.